scripts/testfiles

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var p = path.resolve('./test/');
+
+fs.readdir(p, function(err, files) {
+  var filenames;
+  if (err) {
+    throw err;
+  }
+
+  filenames = files.map(function(file) {
+    return path.join(p, file);
+  }).filter(function(file) {
+    return fs.statSync(file).isFile() &&
+           file.match(/\.js$/) &&
+           !(file.match('examplesRunner') || file.match('bundle.js'));
+  }).join(' ');
+  process.stdout.write(filenames);
+});
```

Comment:
This script could be replaced with

    find test -type f -depth 1 -name '*.js' -not -name bundle.js -not -name test.examplesRunner.js

were in not for Windows. :\